### PR TITLE
Feat/signup

### PR DESCRIPTION
**Pull request by SheldonGomel**
1. [Task](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/train-a/sign-up.md)

 2. Screecshot
image

### CODE REVIEW

File: train-A-app/src/app/auth/components/signup/signup.component.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    }
+    @if (unknownServerError) {
+      <p class="form__error">Unknown server error: {{ this.signupForm.errors?.['serverError'] }}</p>
+    }
+    <div class="form__controls">
+      <button mat-flat-button [disabled]="disabledForm" type="submit">Register</button>
+      <a class="form__link" routerLink="/signin" routerLinkActive="active">Sign In</a>
+    </div>
+  </form>
+  <!-- <pre>
```
Comment (Mariia22) [inline train-A-app/src/app/auth/components/signup/signup.component.html]: we need this comment code?

File: train-A-app/src/app/auth/components/signup/signup.component.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+@import '@styles/common.scss';
+
+.signup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $primary;
+  height: 95vh;
+
+  &__form {
```
Comment (Mariia22) [inline train-A-app/src/app/auth/components/signup/signup.component.scss]: I add this code and responsive width to common file. I think you should remove it

**Review by Mariia22 — APPROVED** [review]

File: train-A-app/src/app/auth/components/signup/signup.component.scss (hunk 2 above)
Comment (SheldonGomel) [reply train-A-app/src/app/auth/components/signup/signup.component.scss]: background-color became white

Comment (Mariia22) [reply train-A-app/src/app/auth/components/signup/signup.component.scss]: Yep, I've changed it

**Pull request merged**